docker/docker-compose.yml

```diff
@@ -357,7 +357,39 @@ services:
         condition: service_started
       keycloak:
         condition: service_started
+      notify:
+        condition: service_started
 
+  notify:
+    healthcheck:
+      test: ["CMD", "java", "/app/scripts/HealthCheck.java", "http://localhost:8080/actuator/health", "||", "exit", "1"]
```

Comment:
Just incase you didn't know - this file doesn't exist in notify at present 🙂

Reply:
awesome!